armory/utils/shape_gen.py

```diff
@@ -0,0 +1,410 @@
+# To add a new shape, add its name and function to _SHAPES
+from dataclasses import dataclass
+import inspect
+from io import BytesIO
+import os
+from pathlib import Path
+from typing import Union
+
+from PIL import Image
+import cairosvg
+import matplotlib.pyplot as plt
+import numpy as np
+import svg
```

Comment:
Which subgroup should cairosvg and svg.py be added to in the pyproject.toml for code that will be potentially called during an armory run? Should this be in engine or pytorch? Which subgroup gets installed on the base docker image?